CondFormats/EcalObjects/BuildFile.xml

```diff
@@ -3,13 +3,18 @@
 <use name="DataFormats/EcalDetId"/>
 <use name="DataFormats/EcalDigi"/>
 <use name="DataFormats/Math"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SoATemplate"/>
 <use name="HeterogeneousCore/CUDACore"/>
 <use name="HeterogeneousCore/CUDAUtilities"/>
 <use name="boost"/>
 <use name="boost_serialization"/>
 <use name="rootmath"/>
 <use name="clhep"/>
 <use name="cuda"/>
+<use name="HeterogeneousCore/AlpakaCore"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
This should be
```suggestion
<flags ALPAKA_BACKENDS="cuda rocm"/>
```
or (in a recent IB)
```suggestion
<flags ALPAKA_BACKENDS="!serial"/>
```

Reply:
Rebased to yesterday's IB and implemented the second option.